integration/qbft/scenarios/integration_test_type.go

```diff
@@ -99,7 +96,7 @@ func (it *IntegrationTest) bootstrap(ctx context.Context) (*scenarioContext, err
 
 	forkVersion := protocolforks.GenesisForkVersion
 
-	ln, err := p2pv1.CreateAndStartLocalNet(ctx, loggerFactory, forkVersion, len(it.OperatorIDs), len(it.OperatorIDs)-1, false)
+	ln, err := p2pv1.CreateAndStartLocalNet(ctx, loggerFactory, forkVersion, len(it.OperatorIDs), len(it.OperatorIDs)/2, false)
```

Comment:
Could you please change it to `len(it.OperatorIDs)-1`? It was changed on `integration-tests` before merging to increase stability of integration tests in the CI